core/src/main/java/org/elasticsearch/action/bulk/byscroll/DeleteByQueryRequest.java

```diff
@@ -111,4 +111,16 @@ public IndicesOptions indicesOptions() {
         assert getSearchRequest() != null;
         return getSearchRequest().indicesOptions();
     }
+
+    public String[] types() {
+        assert getSearchRequest() != null;
+        return getSearchRequest().types();
+    }
+
+    public IndicesRequest types(String...types) {
```

Comment:
can you return DeleteByQueryRequest here please? Also add a whitespace between `String...` and `types`?